test/name_tests.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
Was this file accidentally committed?

Reply:
There goes my attempt at smuggling ẞ into the codebase... Will remove